apps/vr-tests-react-components/src/stories/Button/Button.stories.tsx

```diff
@@ -0,0 +1,200 @@
+  <Button id={buttonId} appearance="primary" disabled>
+    Hello, world
+  </Button>
+);
+
+export const PrimaryDisabledHighContrast = getStoryVariant(PrimaryDisabled, HIGH_CONTRAST);
+export const PrimaryDisabledDarkMode = getStoryVariant(PrimaryDisabled, DARK_MODE);
+
+export const SubtleDisabled = () => (
+  <Button id={buttonId} appearance="subtle" disabled>
+    Hello, world
+  </Button>
+);
+
+export const SubtleDisabledHighContrast = getStoryVariant(SubtleDisabled, HIGH_CONTRAST);
+export const SubtleDisabledDarkMode = getStoryVariant(SubtleDisabled, DARK_MODE);
+
+export const TransparentDisabled = () => (
+  <Button id={buttonId} appearance="transparent" disabled>
+    Hello, world
+  </Button>
+);
+
+export const TransparentDisabledHighContrast = getStoryVariant(TransparentDisabled, HIGH_CONTRAST);
+export const TransparentDisabledDarkMode = getStoryVariant(TransparentDisabled, DARK_MODE);
+
+export const SizeSmall = () => (
+  <Button id={buttonId} icon={<CalendarMonth />} size="small">
+    Hello, world
+  </Button>
+);
+
+SizeSmall.storyName = 'Size small';
+
+export const SizeLarge = () => (
+  <Button id={buttonId} icon={<CalendarMonth />} size="large">
+    Hello, world
+  </Button>
+);
+
+SizeLarge.storyName = 'Size large';
+
+export const SizeSmallWithLongTextWrapping = () => {
+  const styles = useStyles();
+  return (
+    <Button id={buttonId} className={styles.longText} icon={<CalendarMonth />} size="small">
+      Long text wraps after it hits the max width of the component
+    </Button>
+  );
+};
+
+SizeSmallWithLongTextWrapping.storyName = 'Size small - with long text wrapping';
+
+export const SizeMediumWithLongTextWrapping = () => {
+  const styles = useStyles();
+  return (
+    <Button id={buttonId} className={styles.longText} icon={<CalendarMonth />} size="medium">
+      Long text wraps after it hits the max width of the component
+    </Button>
+  );
+};
+
+SizeMediumWithLongTextWrapping.storyName = 'Size medium - with long text wrapping';
+
+export const SizeLargeWithLongTextWrapping = () => {
+  const styles = useStyles();
+  return (
+    <Button id={buttonId} className={styles.longText} icon={<CalendarMonth />} size="large">
+      Long text wraps after it hits the max width of the component
+    </Button>
+  );
+};
+
+SizeLargeWithLongTextWrapping.storyName = 'Size large - with long text wrapping';
+
+export const WithIconBeforeContent = () => (
+  <Button id={buttonId} icon={<CalendarMonth />}>
+    Hello, world
+  </Button>
+);
+
+WithIconBeforeContent.storyName = 'With icon before content';
+
+export const WithIconBeforeContentRTL = getStoryVariant(WithIconBeforeContent, RTL);
+
+export const WithIconAfterContent = () => (
+  <Button id={buttonId} icon={<CalendarMonth />} iconPosition="after">
+    Hello, world
+  </Button>
+);
+
+WithIconAfterContent.storyName = 'With icon after content';
+
+// export const WithIconAfterContentRTL = getStoryVariant(WithIconAfterContent, RTL);
```

Comment:
The `hover` test for this was flaky and not hovering over the button which caused an incorrect screener change - commenting out for now

Reply:
no longer an issue